CONTRIBUTING.md

```diff
@@ -108,6 +108,27 @@ resulting ``_build`` directory it produces; ``index.html`` is the main page.
 **NOTE:** You can try to use the `make` commands on Windows if you have `make` installed and something like `Cygwin`, `MSYS2` or `MYSYS`, or by using Windows Subsystem for Linux (`WSL`).
 
 
+
+## Configuring Netlify
+
+The steps to configure Netlify to preview pull requests are the following:
```

Comment:
I was a little confused...where do these steps go, exactly? Just copying and pasting the exact literals into some field on the Netlify site, or what?

Reply:
Yes, they need to be declared in the Netlify site.